libs/account/src/development/store.ts

```diff
@@ -28,13 +28,8 @@ async function initDevWallets() {
 }
 initDevWallets();
 
-export const getStoredAccountData = (address: string) => ({
-  ..._devAccounts[address],
-  // We don't store original wallets, so just
-  // strip the signer from the data to remove its privateKey
-  signer: { _isRemote: false }
```

Comment:
Updated validity tests in GDrive storage threw if we were missing an actual signer, and there seems to be no reason not to implement this properly (ie - just by cloning the object we emulate localStorage)